node/pkg/common/chainlock.go

```diff
@@ -35,6 +34,10 @@ type MessagePublication struct {
 	Unreliable bool
 }
 
+func (msg *MessagePublication) TxIDString() string {
+	return "0x" + hex.EncodeToString(msg.TxID)
```

Comment:
Does this make sense given that the goal of this TxID change is to grow to encompass Solana-based IDs as well?

Maybe this method should be renamed to `EVMString()`?

Reply:
Related, it might be nice to have a `SolanaString()` function that does base58 encoding, if appropriate.

Reply:
I definitely don't want to have `EVMString` and `SolanaString` because we have many other chains. Right now, those all mangle their identifiers into `ethCommon.Hash` for historical reasons. I don't want to risk breaking other chains with this PR, so I'm leaving them as is. However, going forward, they could be changed to use something more relevant to each protocol.

Since the field is a byte array, logging it as hex seems like the safest approach. I add the "0x" on the front to keep it consistent with what it previously was.

Does all this make sense?

Reply:
Sure, makes sense